.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Site
+on:
+  push:
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Is there a reason to use this old version?

Reply:
No reason, I just copied it from `release.yml`